Library/Tracking/KSRAnalytics.swift

```diff
@@ -566,12 +566,22 @@ public final class KSRAnalytics {
 
   // MARK: - Application Lifecycle
 
-  /// Called when the user taps on a TabBarItem
-  public func trackTabBarClicked(_ tabBarItemLabel: TabBarItemLabel) {
+  /**
+   Called when the user selected a tab bar item.
+
+   - parameter tabBarItemLabel: The tab the user is navigating to.
+   - parameter prevTabBarItemLabel: The tab the user is navigating from.
+   */
+  public func trackTabBarClicked(
```

Comment:
Seems only returning a `pageContext` for `.discovery` case, maybe change the name here to reflect that?

Reply:
Good point although technically the method is still called each time a tab bar is tapped and truthfully this could change in the near future.